PC/config.c

```diff
@@ -43,10 +43,14 @@ extern PyObject* PyInit__collections(void);
 extern PyObject* PyInit__heapq(void);
 extern PyObject* PyInit__bisect(void);
 extern PyObject* PyInit__symtable(void);
+#if !defined(MS_WINDOWS) || defined(MS_WINDOWS_DESKTOP) || defined(MS_WINDOWS_GAMES)
```

Comment:
```suggestion
#if defined(MS_WINDOWS_DESKTOP) || defined(MS_WINDOWS_GAMES)
```

We can assume `MS_WINDOWS` inside anything in the `PC` directory

Reply:
Also, this should work under `MS_WINDOWS_SYSTEM` as well, right?

Reply:
actually it does not right now. From @eryksun review yesterday:

> We can't support the system API partition until mmap is updated to call CreateFileMappingW() and OpenFileMappingW(). As of 10.0.19041, "winbase.h" declares the ANSI versions only for desktop and games.

However I guess this is probably not to hard to update, so it does work on there

Reply:
Ah okay, fair enough I guess. I'm fine to just leave a comment here saying that it requires changing the `*A` APIs to `*W` variants. Going in and changing it might be more complicated than it's worth